www/include/common/javascript/datepicker/localizedDatepicker.js

```diff
@@ -204,6 +207,27 @@ function updateEndTime() {
     }
 }
 
+/**
+ * Used for the end DATEPICKER, to avoid an end date value lesser than the start date
+ * Updates the end date according to the start values.
+ */
+function checkEndDate() {
+    let start = moment($('[name="alternativeDateStart"]').val()
+        + ' ' + $(".timepicker").first().val(), "MM/DD/YYYY HH:mm");
+    let end = moment($('[name="alternativeDateEnd"]').val()
+        + ' ' + $(".timepicker").last().val(), "MM/DD/YYYY HH:mm");
+
+    if (start.isSameOrAfter(end)) {
+        turnOffEvents();
+        start.add($('#duration').val(), $('#duration_scale').val());
+        $(".datepicker").last().datepicker("setDate", start.format($(".datepicker").last().datepicker(
+            "option",
+            "dateFormat"
+        ).toUpperCase().replace(/Y/g, 'YY')));
```

Comment:
```suggestion
        $(".datepicker").last()
            .datepicker(
                "setDate",
                start.format(
                    $(".datepicker").last().datepicker("option", "dateFormat").toUpperCase().replace(/Y/g, 'YY')
                )
            );
```

Reply:
Format of start date and end date is not the same ? (YY | YYYY)

Reply:
The format of moment and returned datepicker value are not the same.